scss/index/_popUp.scss

```diff
@@ -0,0 +1,104 @@
+// Animation @keyframes
+@keyframes fade-in {
+    from {
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
+//background opacity
+.index.active{
+    opacity: 0.90;  
+}
+
+//active class will be removed when pop up is closed
+.pop-up-wrapper.active{
+    z-index: 1;
+}
+
+// Desktop Size
+//pop-up wrapper
+.pop-up-wrapper {
+    position: fixed;
+    top: 20%;
+    left: 8%;
+    right: 8%;
+    z-index: -1 ;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    animation-name: fade-in;
+    animation-duration: 0.5s;
+    
+
+    //pop-up box
+    .popup-overlay .active-popup {
+        visibility: visible;
+        text-align: center;
+        width: 50%;
+        height: 400px;
+        margin: 0 auto;
+        background-color:rgb(145, 106, 64);
+        display:flex;
+
+
```

Comment:
Why do some of these have double white space while some only single white space.